code/modules/admin/verbs/adminhelp.dm

```diff
@@ -15,22 +15,15 @@ var/list/adminhelp_ignored_words = list("unknown","the","a","an","of","monkey","
 	if(prefs.muted & MUTE_ADMINHELP)
 		to_chat(src, "<span class='red'>Error: Admin-PM: You cannot send adminhelps (Muted).</span>")
 		return
-	if(src.handle_spam_prevention(msg,MUTE_ADMINHELP))
-		return
-
-	/**src.verbs -= /client/verb/adminhelp
-
-	spawn(1200)
-		src.verbs += /client/verb/adminhelp	// 2 minute cool-down for adminhelps
-		src.verbs += /client/verb/adminhelp	// 2 minute cool-down for adminhelps//Go to hell
-	**/
-
+	var/msg = input(usr, "What message would you like to adminhelp?", "Send a message to admins", "")
```

Comment:
This is worded awkwardly.

Reply:
Give me a better one

Reply:
* "What do you need help with?"
* "What message would you like to send to the administrators?"
* "What is the nature of your emergency?"

Reply:
I'll go for 2, thanks